cf_xarray/accessor.py

```diff
@@ -376,7 +393,8 @@ def _get_with_standard_name(
     "dims_dict": (_get_axis_coord,),  # swap_dims, rename_dims
     "shifts": (_get_axis_coord,),  # shift, roll
     "pad_width": (_get_axis_coord,),  # shift, roll
-    # "names": something_with_all_valid_keys? # set_coords, reset_coords
+    "names": (_get_with_key,),  # set_coords, reset_coords, drop_vars
```

Comment:
I think just `(_get_axis_coord, _get_measure, _get_with_standard_name)` will work now.

Reply:
Yup, you're right!